Report/Sections/Abstract.tex

```diff
@@ -1 +1 @@
-Pedestrians trajectories prediction has become a popular topic in recent years. Many approaches can be used to accomplish the prediction, including deterministic and non-deterministic ways. In this report we conduct research and discuss how to use a non-deterministic method, i.e. deep learning methods to solve the prediction problem. The algorithm we are going to use is a combination of SocialGAN and InfoGAN. Firstly, we train a model by the adopted algorithms. Secondly, during the research, we propose our research questions and try to answer them in the report.
\ No newline at end of file
+Pedestrians trajectory prediction has received significant importance in recent years due to the increase in autonomous systems, specifically in the automotive industry because of the rising number of self-driving vehicles. Different methods can be used to accomplish the trajectory prediction task, but deterministic and non-deterministic methods are the most commonly used methods. In this report, we used the non-deterministic method, for trajectory prediction and tried to overcome this problem by using sequence prediction and InfoGAN (an information-theoretic extension of Generative Adversarial Network): our recurrent sequence-to-sequence model observes the small subset of pedestrian trajectory as history and predicts the future trajectory, to predict it uses a pooling mechanism to aggregate the information across the other nearby trajectories in the dataset. Our InfoGAN implementation for trajectory prediction learns and predicts disentangled representations of plausible future trajectories in a completely unsupervised manner. We show through experiments, that our proposed method can generate more diverse trajectories because of the disentangled representations.
```

Comment:
What do you mean by 'predicts disentangled representations of plausible future trajectories'? The disentangled factors is something we try to get from the experiments, we don't need to predict it. 
Also I think out goal is to disentangle the factors, not generate more diverse trajectories

Reply:
Don't we need to tell what we accomplished exactly from the experiments?

Reply:
> What do you mean by 'predicts disentangled representations of plausible future trajectories'? The disentangled factors is something we try to get from the experiments, we don't need to predict it. Also I think out goal is to disentangle the factors, not generate more diverse trajectories

if we have disentangled representation, or we can achieve disentanglement via our model then we can generate more trajectories. Right? 

if the sentence or the concept does not feel right we can improve it or just remove it. Or just simply discard it.

Reply:
> Don't we need to tell what we accomplished exactly from the experiments?

"We show through experiments, that our proposed method can generate more diverse trajectories because of the disentangled representations."

I think this part shows that we were able to disentangle and hence because fo that we are able to generate more trajectories. I do not think that we should write here that we were only able to disentangle the direction only. 

May be we can write it as "Through our experiment we were able to disentangle one factor of the trajectory via the latent code, which is the direction of the pedestrian" or "Our experiment tried to disentangle multiple factor of the trajectory using the latent codes, however we were only successful with the direction of the trajectory/pedestrian".

Reply:
> 

It should be 'learn disentanglement' and 'predict trajectories.' You can't predict 'disentanglement'

Reply:
"disentangled representations" does not mean learn or predict disentanglement. It actually means that it learns the trajectory disentangle it and generate new.